dev/app.js

```diff
@@ -5,7 +5,14 @@ var search = instantsearch({
   appId: 'latency',
   apiKey: '6be0576ff61c053d5f9a3225e2a90f76',
   indexName: 'instant_search',
-  urlSync: window.history && 'pushState' in window.history ? true : {useHash: true}
+  urlSync: {
+    useHash: !(window.history && 'pushState' in window.history),
+    mapping: {
+      q: 'query',
+      hPP: 'hits',
+      hFR: 'hierarchical'
+    }
+  }
```

Comment:
Changes here only for demo purposes. We can remove them as soon as it's reviewed.